4b/Grains/src/Coordinator.cs

```diff
@@ -25,11 +41,28 @@ public async Task<string> RunTests()
                 GrainFactory.GetGrain<ITest2>(8).HelloWorldTest()
             };
 
+            testRun.Times.Start = DateTime.Now.ToString(CultureInfo.InvariantCulture);
```

Comment:
Let's always use the current utc time. Also nice work around `ToString` with current culture. This is a sample xml I got from the pipeline to help you with the formatting:

```xml
<?xml version="1.0" encoding="utf-8"?>
<TestRun    id="7e11beb4-9d63-49a6-a1d1-1ce5f146c994"
            name="@6a10d6b4491e 2022-01-14 15:54:48"
            xmlns="http://microsoft.com/schemas/VisualStudio/TeamTest/2010">
  <Times    creation="2022-01-14T15:54:48.3843255+00:00"
            queuing="2022-01-14T15:54:48.3843256+00:00"
            start="2022-01-14T15:54:46.0783493+00:00"
            finish="2022-01-14T15:54:59.7952314+00:00" />
  <TestSettings name="default"
                id="8f5a0bc6-3b28-4d2c-8a34-454bfc9cd26c">
    <Deployment runDeploymentRoot="_6a10d6b4491e_2022-01-14_15_54_48" />
  </TestSettings>
```